service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -79,4 +79,5 @@
     @Valid
     private final List<@NotNull(message = "You need to add details to children")Element<Child>> children1;
     private final List<Element<HearingBooking>> hearingDetails;
+    private final List<Element<Recipients>> statementOfService;
```

Comment:
Personally I think the name of variable confusing - we have got type of recipients but the field is statementOfService?

Reply:
i agree I think it should be a List of recipients